x-pack/test/functional/apps/cloud_security_posture/benchmarks.ts

```diff
@@ -0,0 +1,213 @@
+         * This is the first installation of the integration.
+         * other test assume it is installed
+         */
+        packageInfo = await getPackageInfo();
+        basePackagePolicy = await installKSPM(id());
+      });
+
+      it('navigates to CSP Benchmarks page', async () => {
+        await cloudPosture.benchmarks.goToBenchmarkPage();
+        await cloudPosture.benchmarks.assertBenchmarkPageExists();
+      });
+
+      it('navigates from CSP Benchmarks page to Fleet Add Integration page', async () => {
+        await cloudPosture.benchmarks.goToBenchmarkPage();
+        await cloudPosture.benchmarks.goToNewIntegrationPage();
+        // TODO: add assertion
+      });
+
+      it('navigates from CSP Benchmarks page to Fleet Agent Policy page', async () => {
+        await cloudPosture.benchmarks.goToBenchmarkPage();
+        await cloudPosture.benchmarks.goToIntegrationAgentPolicyPage();
+        // TODO: add assertion
+      });
+    });
+
+    describe('Table Data', () => {
+      /** Tests below use integration 'csp-00' (first installed) */
+
+      it('displays integration name column', async () => {
+        await cloudPosture.benchmarks.goToBenchmarkPage();
+        await cloudPosture.benchmarks.assertTableColumnValueExists(
+          'benchmarks-table-column-integration',
+          'csp-00'
+        );
+      });
+
+      it('displays integration type column', async () => {
+        await cloudPosture.benchmarks.assertTableColumnValueExists(
+          'benchmarks-table-column-integration_type',
+          'Kubernetes Security Posture Management'
+        );
+      });
+
+      it('displays agent policy name column', async () => {
+        await cloudPosture.benchmarks.assertTableColumnValueExists(
+          'benchmarks-table-column-agent-policy',
+          getAgentPolicyName('csp-00')
+        );
+      });
+
+      it('displays agents count column', async () => {
+        await cloudPosture.benchmarks.assertTableColumnValueExists(
+          'benchmarks-table-column-number-of-agents',
+          '0' // No agents should be enrolled
+        );
+      });
+
+      it('displays created by column', async () => {
+        await cloudPosture.benchmarks.assertTableColumnValueExists(
+          'benchmarks-table-column-created-by',
+          basePackagePolicy.created_by // TODO: create user explicity
+        );
+      });
+
+      it.skip('created time column is displayed', async () => {
+        // UI is formatted.
+        // TODO: add 'title' timestamp to the column and assert on it?
+      });
+    });
+
+    describe('Table Pagination', async () => {
+      before(async () => {
+        /**
+         * We need more than 10 rows to test pagination, and we currently have 1
+         * unfortunately, this is what takes most of the time in all of the tests
+         *
+         * TODO: find a way to speed this up.
+         */
+        await Promise.all(Array.from({ length: 10 }, () => installKSPM(id())));
+      });
```

Comment:
to test pagination, we need more integrations installed than the page size. 
there's 2 issues here: 
 - currently, table page size is fixed at `10 | 25 | 50`, and is not controlled by URL, so we can't change it to 2, for example. so here we're creating 10 more integrations, total of 11 installed, so we do the pagination test on page size of `10` and `25` 
  
 - we need to create N integrations: 
    - via UI - mega slow and wasteful as we're not trying to test installation here
    - via API: faster but still slow and wasteful, same reason ^  
    - via archive 
         - existing archive: `x-pack/test/functional/es_archives/fleet/agents` comes with only 2 agent policies, we'd still need to create package policies.  
        - new archive - probably the best option, still figuring this out 😄